samples/linkedlists/linkedList.py

```diff
@@ -0,0 +1,101 @@
+#!/bin/python3
```

Comment:
Can we move the file to this folder of commons
`https://github.com/csklub/weekly-sessions/tree/master/samples/commons/python`

Reply:
Sure, that is fine.

Reply:
Hi @ezkemboi , I already moved the file, following is a refactor, with the proposed changes.